doc/examples/simulate_dki.py

```diff
@@ -0,0 +1,83 @@
+"""
+==========================
+DKI MultiTensor Simulation
+==========================
+
+In this example we show how someone can simulate the diffusion tensor,
+diffusion kurtosis tensor and signals of a single voxel by taking into account 
+different cellular media using a multi-compartment simulatations. Simulations 
+are preformed according to: 
+
+[1] R. Neto Henriques et al., "Exploring the 3D geometriy of the diffusion 
+    kurtosis tensor - Impact on the development of robust tractography 
+    procedures and novel biomarkers", NeuroImage (2015) 111, 85-99.
+"""
+
+import numpy as np
+from dipy.sims.voxel import multi_tensor_dki
+
+"""
+For the simulation we will need a GradientTable with the b-values and b-vectors
+Here we use the one we created in :ref:`example_gradients_spheres`.
+"""
+
+from gradients_spheres import gtab
```

Comment:
Yes. I use this example also to keep consistency to the simulations shown in simulate.py.